src/mds/journal.cc

```diff
@@ -2132,11 +2132,9 @@ void EUpdate::replay(MDSRank *mds)
 	       << " < " << cmapv << dendl;
       // open client sessions?
       map<client_t,entity_inst_t> cm;
-      map<client_t, uint64_t> seqm;
       bufferlist::iterator blp = client_map.begin();
       ::decode(cm, blp);
-      mds->server->prepare_force_open_sessions(cm, seqm);
-      mds->server->finish_force_open_sessions(cm, seqm);
+      mds->sessionmap.open_sessions(cm);
```

Comment:
I don't understand why this change is being made at all. I think maybe the commit means you want to switch to doing this, but I'm not sure why. It looks like you're just tossing out a bunch of asserts and other safety/metadata code.